File: src/main/java/com/tolanguage/tolanguage/TolanguageApplication.java
package com.tolanguage.tolanguage;

import com.tolanguage.tolanguage.config.property.ApplicationProperty;
import com.tolanguage.tolanguage.config.property.CacheProperty;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.context.properties.EnableConfigurationProperties;

@SpringBootApplication
@EnableConfigurationProperties({ApplicationProperty.class, CacheProperty.class})
public class TolanguageApplication {

    public static void main(String[] args) {
        SpringApplication.run(TolanguageApplication.class, args);
    }

}

File: src/main/kotlin/com/tolanguage/TolanguageApplication.kt
package com.tolanguage

import com.tolanguage.config.property.ApplicationProperty
import com.tolanguage.config.property.CacheProperty
import com.tolanguage.config.property.MailServiceProperty
import com.tolanguage.config.property.SessionProperty
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.context.properties.EnableConfigurationProperties
import org.springframework.boot.runApplication

@SpringBootApplication
@EnableConfigurationProperties(
    ApplicationProperty::class,
    CacheProperty::class,
    SessionProperty::class,
    MailServiceProperty::class
)
class TolanguageApplication

fun main(args: Array<String>) {
    runApplication<TolanguageApplication>(*args)
}
